### PR TITLE
Update Python Content

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
EDIT: All changes are moved to PR per Folder in both Python courses

Move single-line commands from multiple lines back to a single line as we now wrap code blocks

Also fix incorrect indentation in some insights. Example:
```python
# one insight would have this indentation
class Bird(Animal):
    def identify(self):
         # added line, calls parent method
        super().identify()
        print("I am a bird")
```

Where the comment is slightly more indented and the method and its code are indented with 4 spaces, then in the following insight (in some case the very same insight) the indentation would look like this:
```python
class Bird(Animal):
  def identify(self):
    # added line, calls parent method
    super().identify()
    print("I am a bird")
```

which is only 2 spaces.

I am going to go for the 2 spaces one as that would make less wrapping between code blocks

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| distinguish-the-mutability-of-common-types | <ul><li>`PYTHON OBJECTS: MUTABLE VS. IMMUTABLE` pointing to `https://codehabitude.com/2013/12/24/python-objects-mutable-vs-immutable/` is broken because `HTTP_404`</li></ul> |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| distinguish-the-mutability-of-common-types | <ul><li>`PYTHON OBJECTS: MUTABLE VS. IMMUTABLE` pointing to `https://codehabitude.com/2013/12/24/python-objects-mutable-vs-immutable/` is broken because `HTTP_404`</li></ul> |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Review by nem035 — DISMISSED** [review]
:muscle:

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| distinguish-the-mutability-of-common-types | <ul><li>`PYTHON OBJECTS: MUTABLE VS. IMMUTABLE` pointing to `https://codehabitude.com/2013/12/24/python-objects-mutable-vs-immutable/` is broken because `HTTP_404`</li></ul> |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| distinguish-the-mutability-of-common-types | <ul><li>`PYTHON OBJECTS: MUTABLE VS. IMMUTABLE` pointing to `https://codehabitude.com/2013/12/24/python-objects-mutable-vs-immutable/` is broken because `HTTP_404`</li></ul> |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/functional-programming/arrays-i/the-slice-built-in-function.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/arrays-i/the-slice-built-in-function.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/arrays-i/the-zip-built-in-function.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/arrays-i/the-zip-built-in-function.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/arrays-ii/the-map-built-in-function.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/arrays-ii/the-map-built-in-function.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/arrays-ii/the-sorted-built-in-function.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/arrays-ii/the-sorted-built-in-function.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/comprehension/dictionary-comprehension.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/comprehension/dictionary-comprehension.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/comprehension/nested-lists-comprehension.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/comprehension/nested-lists-comprehension.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/comprehension/set-comprehension.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/comprehension/set-comprehension.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/comprehension/speed-up-your-for-loop-using-map-or-list-comprehensions.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/comprehension/speed-up-your-for-loop-using-map-or-list-comprehensions.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/decorators/decorators-methods.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/decorators/decorators-methods.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/decorators/decorators-syntax.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/decorators/decorators-syntax.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/decorators/functools-wraps.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/decorators/functools-wraps.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/decorators/what-are-decorators.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/decorators/what-are-decorators.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/functional-programming-ii/functional-particularities-of-python.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/functional-programming-ii/functional-particularities-of-python.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/functional-programming/what-is-functional-programming.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/functional-programming/what-is-functional-programming.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/generators/generator-of-generators.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/generators/generator-of-generators.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/generators/recursive-generator.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/generators/recursive-generator.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/generators/yield-and-next.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/generators/yield-and-next.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/iterators/the-iteration-protocol.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/iterators/the-iteration-protocol.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/iterators/the-itertools-module-ii.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/iterators/the-itertools-module-ii.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/python-immutability/atomicity-of-failure.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/python-immutability/atomicity-of-failure.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/python-immutability/distinguish-the-mutability-of-common-types.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/python-immutability/distinguish-the-mutability-of-common-types.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/functional-programming/python-immutability/why-types-have-immutability-ii.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/functional-programming/python-immutability/why-types-have-immutability-ii.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/advanced-queues/prioritize-your-queue.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/advanced-queues/prioritize-your-queue.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/advanced-queues/queue-s-and-threads.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/advanced-queues/queue-s-and-threads.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/advanced-referencing/context-manager-types-with.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/advanced-referencing/context-manager-types-with.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/advanced-referencing/weakref-proxies.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/advanced-referencing/weakref-proxies.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/basic-file-manipulation/how-to-open-a-file-object.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/basic-file-manipulation/how-to-open-a-file-object.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/basic-file-manipulation/writing-files.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/basic-file-manipulation/writing-files.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/bits-bytes-hexadecimals/bytearray-objects.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/bits-bytes-hexadecimals/bytearray-objects.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/classes-i/class-keywords.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/classes-i/class-keywords.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/classes-i/creating-classes.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/classes-i/creating-classes.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/classes-i/method-overriding.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/classes-i/method-overriding.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/classes-i/using-classes.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/classes-i/using-classes.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/classes-ii/classes-ii-discussion.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/classes-ii/classes-ii-discussion.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/classes-ii/method-objects.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/classes-ii/method-objects.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/classes-ii/private-variables.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/classes-ii/private-variables.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/classes-iii/dynamically-create-types.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/classes-iii/dynamically-create-types.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/classes-iii/special-attributes-of-objects-and-classes.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/classes-iii/special-attributes-of-objects-and-classes.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/control-flow-i/boolean-operators.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/control-flow-i/boolean-operators.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/control-flow-i/if-elif-else-statements.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/control-flow-i/if-elif-else-statements.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/control-flow-i/if-statements.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/control-flow-i/if-statements.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/control-flow-i/indentation-and-commenting.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/control-flow-i/indentation-and-commenting.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/control-flow-i/intro-to-booleans.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/control-flow-i/intro-to-booleans.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/control-flow-ii/the-in-operator.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/control-flow-ii/the-in-operator.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/control-flow-ii/the-not-operator.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/control-flow-ii/the-not-operator.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/deep-into-collections/best-way-to-implement-a-simple-queue.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/deep-into-collections/best-way-to-implement-a-simple-queue.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/deep-into-collections/double-ended-queues-with-deque.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/deep-into-collections/double-ended-queues-with-deque.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/deep-into-collections/enhance-your-tuple-s.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/deep-into-collections/enhance-your-tuple-s.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/intro-to-modules/namespace-and-scoping.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/intro-to-modules/namespace-and-scoping.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/intro-to-modules/the-from-import-statement.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/intro-to-modules/the-from-import-statement.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/is-your-python-healthy/debugging-with-print.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/is-your-python-healthy/debugging-with-print.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/is-your-python-healthy/errors-and-execeptions.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/is-your-python-healthy/errors-and-execeptions.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/is-your-python-healthy/python-debugger-ii.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/is-your-python-healthy/python-debugger-ii.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/looping/break-and-continue-statements.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/looping/break-and-continue-statements.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/looping/for-loops.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/looping/for-loops.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/looping/looping-techniques.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/looping/looping-techniques.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/looping/using-else-in-loops.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/looping/using-else-in-loops.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/looping/while-loops.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/looping/while-loops.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/meet-python/what-is-python.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/meet-python/what-is-python.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/more-on-dictionaries/dictionaries-from-lists.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/more-on-dictionaries/dictionaries-from-lists.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/more-on-dictionaries/keep-things-in-order-with-ordereddict.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/more-on-dictionaries/keep-things-in-order-with-ordereddict.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/more-on-lists/using-lists-as-queues.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/more-on-lists/using-lists-as-queues.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/playing-with-time/playing-with-time.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/playing-with-time/playing-with-time.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/playing-with-time/time-object.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/playing-with-time/time-object.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/python-functions/calling-functions.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/python-functions/calling-functions.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/python-functions/defining-functions.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/python-functions/defining-functions.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/python-functions/nested-functions.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/python-functions/nested-functions.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/python-functions/the-return-statement.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/python-functions/the-return-statement.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/python-recipes/a-simple-way-to-select-a-random-item-from-a-list-tuple-data-stucture.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/python-recipes/a-simple-way-to-select-a-random-item-from-a-list-tuple-data-stucture.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/python-tips/pretty-print-data-structures.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/python-tips/pretty-print-data-structures.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/string-recipes/recipe-to-normalize-text.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/string-recipes/recipe-to-normalize-text.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/string-recipes/regular-expressions.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/string-recipes/regular-expressions.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/testing/doctests.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/testing/doctests.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/testing/mocking-tests.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/testing/mocking-tests.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/testing/nose-testing.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/testing/nose-testing.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/unordered-data-types/dictionary-methods-ii.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/unordered-data-types/dictionary-methods-ii.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/unordered-data-types/dictionary-standard-mapping-type.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/unordered-data-types/dictionary-standard-mapping-type.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/utilities-i/your-own-python-calendar.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/utilities-i/your-own-python-calendar.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/utilities-ii/coroutine-utility-function.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/utilities-ii/coroutine-utility-function.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/utilities-ii/working-with-junk-data.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/utilities-ii/working-with-junk-data.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/working-with-strings/efficient-concatenation-with-join.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/working-with-strings/efficient-concatenation-with-join.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)
- [ ] [enkidevs/curriculum/python/python-core/working-with-strings/three-ways-to-substitute-a-substring-of-a-string.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/working-with-strings/three-ways-to-substitute-a-substring-of-a-string.md?ref=05fbb6dfd366aa70e6a01a521606b8c5e12fb0f3)

File: python/python-core/utilities-ii/working-with-junk-data.md [hunk 1]
```diff
@@ -65,8 +63,7 @@ print(s.find_longest_match(0, 5, 0, 9))
 Complete the `SequenceMatcher`  constructor such that empty spaces are treated as junk:
 
 ```python
-s = SequenceMatcher(??? x: x == ???,
-    “ abcd”, “abcd abcd”)
+s = SequenceMatcher(??? x: x == ???, “ abcd”, “abcd abcd”)
```
Comment (nem035) [inline python/python-core/utilities-ii/working-with-junk-data.md:66]: The quotes here look different?

Comment (Stefan-Stojanovic) [reply python/python-core/utilities-ii/working-with-junk-data.md:66]: The PR that changes the quotes is [this one](https://github.com/enkidevs/curriculum/pull/2977)

**Pull request merged**